clients/shared/services/api/apiMethods.ts

```diff
@@ -0,0 +1,35 @@
+import api from '@web/config/apiConfig'
+import { AxiosResponse } from 'axios'
+
+export const get = async(url: string, params?: object): Promise<AxiosResponse> => {
+  const response = await api.get(url, { params })
+  return response
+}
+
+export const getSingle = async(
+  url: string,
+  id: string | number,
+  params?: object
+): Promise<AxiosResponse> => {
+  const response = await api.get(`${url}/${id}`, { params })
+  return response
+}
+
+export const post = async(url: string, data: object): Promise<AxiosResponse> => {
+  const response = await api.post(url, data)
```

Comment:
honestly you can just
`return api.post....`

infact 

```tsx
export const post = async(url: string, data: object): Promise<AxiosResponse> => api.post(url, data)
```

Reply:
its giving linting error if we don't use await